iModelCore/GeoCoord/BaseGeoCoord/basegeocoord.cpp

```diff
@@ -8946,7 +8946,7 @@ struct GeoCoordWorkspaces {
             Logging::LogMessageV("GeoCoord", LOG_ERROR, "Unable to read data for GCS file %s from workspace %s, rc=%d", path, row.db->GetDbFileName(), rc);
             delete resource; // weren't able to read from row.
         }
-        Logging::LogMessageV("GeoCoord", LOG_WARNING, "Unable to find GCS file %s in any workspace", path);
+        Logging::LogMessageV("GeoCoord", LOG_INFO, "Unable to find GCS file %s in any workspace", path);

```

Comment:
Should this log messaged be removed entirely in favor of the later message stating if the file was not found in a workspace or locally?

Reply:
A bit nitpicky but, when we log 'in workspace or locally' are we actually searching locally always? The log message implies that we checked both, but I imagine we could fail to have it in our workspace and then we'd log 'in workspace or locally' without searching locally right?

Reply:
In the case of that particular log message, we are checking both. It failed to find anything in a workspace, so it looks locally, if it finds it in neither than that msg is logged

Reply:
Sounds good,

Regarding your original question: Should this log messaged be removed entirely in favor of the later message stating if the file was not found in a workspace or locally?

We should keep the log message but change warning to info and vice versa based off of if loadLocalFiles is true. If loadlocalFiles is false we'd log nothing at all if we removed the message.